src/esp/bindings/GfxBindings.cpp

```diff
@@ -83,20 +84,26 @@ void initGfxBindings(py::module& m) {
       .def(
           "draw",
           [](Renderer& self, sensor::VisualSensor& visualSensor,
-             scene::SceneGraph& sceneGraph, RenderCamera::Flag flags) {
-            self.draw(visualSensor, sceneGraph, RenderCamera::Flags{flags});
+             scene::SceneGraph& sceneGraph, RenderCamera::Flags flags) {
```

Comment:
@aclegg3 : 
Would you mind taking a close look at it? 
The original code was introduced in #679 . I believe it is a bug. Can you help me to double verify it?
Thanks.